### ISSUE TITLE
Archiv crashes shiori

### ISSUE THREAD

**Issue by rmayergfx** [go-shiori/shiori#689]
## Data
- **Shiori version**: v1.5.5
- **Database Engine**: 
- **Operating system**: Docker Synology
- **CLI/Web interface/Web Extension**: 0.8.7

## Describe the bug / actual behavior
Doing a bookmark with archive with web extension:
URL to test:
https://kb.synology.com/de-de/DSM/tutorial/Which_Synology_NAS_supports_WDDA

Readable version is fine, klick on "View Archive" ist not opening local archive, tooks very long time and on the status line you see that the system is trying to connect to kb.synology.com and when the page finaly is loaded you the the Cookie Settings and a wired page with missing formatting.

![grafik](https://github.com/go-shiori/shiori/assets/105392164/e5aef0dc-0e60-4328-b7fe-9f3d9c050964)

**Comment by fmartingr** [comment]
Thanks for the report, commenting on the two issues you experienced:

1. Regarding the slowness: In my case it didn't crash Shiori at all, it just took **very long** to archive because some resources come from a server that is so slow that times out after a while:

```
# From obelisk output:
WARN[0065] download failed: Get "http://kb.synology.com:8080/helpfile/": dial tcp 52.222.144.109:8080: i/o timeout
```

2. Regarding the archive result, since their page have the cookie banner pop up on top of everything, so when you archive the result would be the same, though our archive doesn't store/display styles. 
  
    I tried an archive using [obelisk](https://github.com/go-shiori/obelisk) (the archiver that will replace the current one) and while the styles are ok, the content is not shown (and it took a minute and a half to archive):

    ![Screenshot 2023-08-05 at 09 08 50](https://github.com/go-shiori/shiori/assets/812088/f47ba8f8-98fa-4271-9820-025f05a870e0)

---

In summary: there's no action for Shiori to take here.

If you want to help improve the archival of the page, please report it under [obelisk](https://github.com/go-shiori/obelisk) so we can take that into consideration once we implement that here. Maybe we need to add some cookie bypass or something like that.
